spikeextractors/recordingextractor.py

```diff
@@ -252,19 +250,21 @@ def get_channel_locations(self, channel_ids=None):
         for channel_id in channel_ids:
             location = self.get_channel_property(channel_id, 'location')
             locations.append(location)
-        return locations
+        return np.array(locations)
 
-    def set_channel_groups(self, channel_ids, groups):
+    def set_channel_groups(self, groups, channel_ids=None):
```

Comment:
most of the times you want to set the groups for all channels...

Reply:
very dangerous.
This args invertion will break many codes.
I am ok with this but need to be tested in all package and published notebooks.

Reply:
I'm doing it :) It was very annoying...of course I want all channels by default!